src/pages/MyLists/DeleteListModal/container.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import withModal from '../../../hoc/withModal';
+import { deleteMyListError } from '../../../../state/my_lists/selectors';
+import {
+  deleteMyList as deleteMyListAction
+} from '../../../../state/my_lists/actions';
+import DeleteListModal from './component';
+
+class DeleteListModalContainer extends React.Component {
+  deleteList = () => {
+    const { deleteMyList, id, hideModal } = this.props;
+
+    return (
```

Comment:
I'm not sure that you need to return result of `deleteMyList ` invocation.

Reply:
I agree with you. Removed.